tracer/test/Datadog.Trace.Tests/Telemetry/Metrics/common_metrics.json

```diff
@@ -760,11 +760,23 @@
     },
     "instrum.user_auth.missing_user_id": {
       "tags": [
-        "framework"
+        "framework",
+        "event_type"
+      ],
+      "metric_type": "count",
+      "data_type": "events",
+      "description": "The library was unable to obtain a user ID as part of the login success or failure event. This should only be sent if the user login was also unavailable.",
+      "send_to_user": false,
+      "user_tags": []
+    },
+    "instrum.user_auth.missing_user_login": {
```

Comment:
Have you added this one to dd-go (or it's already there?)

Reply:
yes it's been added thank you